cpp/lib/ome/xml/model/primitives/ConstrainedNumeric.h

```diff
@@ -406,6 +408,34 @@ namespace ome
           }
 
           /**
+           * Increment the constrained value by one.  If the new value
+           * fails the constraint check, this will cause an error.
+           *
+           * @returns the new value.
+           */
+          inline ConstrainedNumeric&
+          operator++ ()
+          {
+            ++this->value;
+            check();
+            return *this;
+          }
+
+          /**
+           * Decrement the constrained value by one.  If the new value
+           * fails the constraint check, this will cause an error.
```

Comment:
It's all the tables of inputs, outputs and whether exceptions are thrown or not.